### PR TITLE
Search error page

### PR DESCRIPTION
**Pull request by alexchoiweb**
-Only added for mobile for now, wasn't sure why it was split from desktop in the code

### CODE REVIEW

**Comment by jmensch1** [comment]
hey @alex, looks like the linter got you... can you run `npm run lint` and push the changes? thanks

**Comment by alexchoiweb** [comment]
Ah my old arch-nemesis, the linter
Done, let me know if there is anything else needed

File: client/web/src/components/info/ResultList.jsx [hunk 1]
```diff
@@ -73,32 +74,34 @@ const ResultList = ({ data, toggleCountyInfo, toggleDrawer, close, open }) => {
...
-              {data.jurisdictionData.locations.length === 1 ? (
-                <p>
-                  {data.jurisdictionData.locations.length} drop off location
-                  available in{' '}
-                  <b>
-                    {data.jurisdictionData.name}, {data.stateData.name}
-                  </b>
-                </p>
+              {data.jurisdictionData.locations.length === 0 ? (
+                <SearchError />
```
Comment (jmensch1) [inline client/web/src/components/info/ResultList.jsx:78]: interesting...i was thinking the error page would be completely separate and have its own url, but i kinda like this way better

**Review by jmensch1 — APPROVED** [review]
looks good

**Pull request merged**